Frontend/DAT257_Website/src/Components/Header/UserLogo.css

```diff
@@ -0,0 +1,19 @@
+.userLogo {
+    width: 60px;
+    height: 60px;
+    margin-left:65vw;
```

Comment:
Not really sure why userLogo has 2 different css instances, one here and one in Header.css, maybe remove one of them? Also, what's the difference between this margin-left and the margin-left in Header.css?